### DISCUSSION TITLE
Updated 6.4 to 6.5 - mailscanner

### DISCUSSION THREAD

**Discussion by Marco31Bike** [YetiForceCompany/YetiForceCRM#17523] [General]
Hi there... mailscanner not reading emails (dont create tickets)
Got following errors:
Warning:  Undefined property: rcube_message_header::$authres_status in /var/www/html/public_html/modules/OSSMail/roundcube/program/actions/mail/index.php on line 553

and

PHP Deprecated:  stripos(): Passing null to parameter #1 ($haystack) of type string is deprecated in /var/www/html/app/Mail/Rbl.php on line 437

Any clue on that?

Best

**Comment by Marco31Bike** [comment]
Maybe you guys can help telling me what mailscanner related code is responsible for connecting to mail folders? Roundcube nicely working, accounts connected to mailscanner, folders selected, but cron scanning and manual scanning  show no scanning result. From Roundcube itself I can create ticket and email is attached to OSSMailView

**Reply by rskrzypczak** [reply]
you probably haven't set any actions in the mail scanner panel.
Remember that you must set appropriate actions and indicate folders to be scanned.
Moreover, in the `Software configuration/Logs/Mail – Download history` panel you can preview recently performed mail scanner operations.

**Reply by Marco31Bike** [reply]
I did. Everything is perfectly configured. Download history empty. When I hit Manual Scanning it showing progress and zero emails checked. Mail server is really polished perfectly. Can you advise which script is directly responsible for fetching emails from mailbox?

**Reply by garryatbrighton** [reply]
Had the same some time ago... it was some mess with how CRM connecting to IMAP. Someone fixed it with dovecot but dunno how. Maybe yeti can share some IMAP server config tricks?

**Reply by Marco31Bike** [reply]
What I detected so far is that yetiforce (for unknown reason) dont get ssl://domain.com:993 (tried actually various settings)
The system itself has no problem (checked from CLI  -openssl etc)
Wonder if here could be some settings which may blocking this connection.
Log shows me:
[2024-07-13 17:43:36] Trying mailbox string: {imap.mydomain.com:143/imap}
[2024-07-13 17:43:37] IMAP connection failed: TLS/SSL failure for imap.mydomain.com: SSL negotiation failed

**Comment by Marco31Bike** [comment]
Further investigation shows that mailscanner doesnt inserts correctly uid to vtiger_ossmailscanner_folders_uid.
All folders uid are "0" No idea where issue is. I have another 6.5 instance but build from scratch. The one with issue is multiple time upgraded but never errors. Essential mailscanner, mail etc modules files are untouched. Dbase config perfect, PHP 8.1

**Reply by mariuszkrzaczkowski** [reply]
there was an error that when the status in the scanner's log table was that scanning was in progress, the scanner was waiting for completion

**Reply by Marco31Bike** [reply]
php8.1 - not working
php8.0 - all OK

So we have issue with php compatibility in 6.5. Any chance to investigate it? Or I am doing something wrong on 3 different installations? (three different servers)

**Comment by rskrzypczak** [comment]
there is no issue 
YF6.5 - works with PHP 7.4, 8.0
YF7 - works with PHP >= 8.1
